gravitee-apim-console-webui/src/services-ngx/api-v2.service.ts

```diff
@@ -128,7 +128,7 @@ export class ApiV2Service {
 
   getLastApiFetch(apiId: string): Observable<Api> {
     if (this.lastApiFetch$.value === null) {
-      return this.get(apiId);
+      this.get(apiId).subscribe();
```

Comment:
I may miss something, but when we have an explicit subscribe we should have an unsubscribe somewhere 🤔

Reply:
I've pushed with the suggestion of @loriepisicchio